build.sbt

```diff
@@ -520,6 +526,19 @@ lazy val discovery = akkaModule("akka-discovery")
   .settings(AutomaticModuleName.settings("akka.discovery"))
   .settings(OSGi.discovery)
 
+lazy val lease = akkaModule("akka-lease")
```

Comment:
Should we make the module name somewhat more general to allow for adding imaginary other related things in the future? Thinking of "akka-consensus" or "akka-coordination"?

Reply:
Good idea. Coordination seems appropriately vauge